### DISCUSSION TITLE
NERVGEAR

### DISCUSSION THREAD

**Discussion by Raywei0214** [1101-GS3073-C/Python-Community-GS3073-C#593] [General]
打這篇文章已經距離nervgear正式發售有一個多月了，想請問各位SAO的好手們，帶帶我這個新手，讓我們一起星爆氣流斬吧，期許我也能像桐人一樣找到后宮
https://www.youtube.com/watch?v=zfln6XMua8M&ab_channel=%E7%98%8B%E7%8B%82%E8%80%81%E7%88%B9

**Comment by RogelioKG** [comment]
摸豆哈雅苦

**Reply by Raywei0214** [reply]
你這樣會有點數嗎我比較好奇這個www

**Reply by awehao** [reply]
那個下面有眼睛，想必是白忙一場ㄌㄡ，我發現我兩個逗號不一樣诶，但都是中文輸入法

**Comment by lukiea** [comment]
![b8ed0f219bd592efebd507fefa6eec4f](https://user-images.githubusercontent.com/113413519/210802944-82e0b2eb-91d9-4ab0-af0a-69c1f261e07a.jpg)

**Comment by Raywei0214** [comment]
看看別人，再想想自己，會發現問題的核心其實就在你身旁。若到今天結束時我們都還無法釐清nervgear的意義，那想必我們昨天也無法釐清。

**Comment by bww0401** [comment]
如果完全潛行真的實現，感覺會引發更多的問題，因此要制定相關法規才行

**Comment by larry40305** [comment]
如果真的實現並發行遊戲，那會把破壞腦部的功能寫進來嗎。不是很多玩家都喜歡能真實回饋的遊戲嗎?
